.travis.yml

```diff
@@ -1,30 +1,34 @@
 sudo: false
 language: python
-
 python:
-  - "2.6"
-  - "2.7"
-  - "pypy"
-  - "3.3"
-  - "3.4"
-  - "3.5"
-
+- '2.6'
+- '2.7'
+- pypy
```

Comment:
the `travis setup pypi` command did re-serialize the file, and pypy is automatically a string